utils/migration.js

```diff
@@ -48,10 +48,43 @@ const migration_2_Metrics = async () => {
     // TODO
 }; */
 
+const migration_4_Themes = async () => {
+    console.log('Applying migration script "migration_4_Themes"...');
+    const theme = await mongoose.connection.collection('themeConfigs').findOne({});
+    console.log(Array.isArray(theme.config.translation.fr));
```

Comment:
Don't do it only for fr, iterate over all possible languages `Object.values(theme.config.translation)`